packages/addons-v5/commands/addons/destroy.js

```diff
@@ -1,40 +1,53 @@
+/* eslint-disable no-await-in-loop */
```

Comment:
This is a pretty good lint rule in general, and I would like to see this line gone. I am curious why this disable wasn't necessary before.
I left work around examples in other comments.

Reply:
I added this only for some files that I was touching just to avoid making this PR more complex to review. There's a lot of linter warnings to be reviewed. I did it at the file level because the intended behavior is to not parallelize the execution of the promises, but to execute them serialized. I agree that the warning might be useful in other contexts as a good indicator of a bad implementation.

Reply:
This makes sense, and was already in the code.